src/encoded/static/components/genetic_modification.js

```diff
@@ -54,73 +54,74 @@ IntroducedTags.propTypes = {
 };
 
 
-// All possible genetic modification object modification site property names. This might need to
-// change if the genetic_modification.json schema changes.
-const modificationSitePropNames = ['modified_site_by_target_id', 'modified_site_by_coordinates', 'modified_site_by_sequence', 'modified_site_nonspecific'];
+const modifiedSiteRenderers = {
```

Comment:
I have not see someone use an object like this in a long time. It was common pattern used before ES6 was widely accepted in forms of Patterns like revealing module pattern (not exactly but the core idea). Nowadays, people would just create a new class. Your keys would be the methods in the class. The value would be the body.

Reply:
Not sure how a class would work for this case. Normally you instantiate a class to make an object containing methods that operate on that object. In this case, this object’s “methods” get used on different things that aren’t necessarily objects, and certainly not connected to methods themselves.